### PR TITLE
close 5 implement user and user controller

### LINKED ISSUE #5: Implement User and User Controller
Implement User object and User Controller for handling necessary endpoints.

### PR DESCRIPTION
**Pull request by Natasha-Zero**
close #5

### CODE REVIEW

**Comment by Natasha-Zero** [comment]
The endpoint still affected by spring security, but will fix later.

File: src/main/java/com/TrackMyWorkout/Server/User/User.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        this.userName = userName;
+        this.passWord = password;
+        this.email = email;
+    }
+
+    public User() {
+        this.userName = null;
+        this.passWord = null;
+        this.email = null;
+        this.verified = false;
```
Comment (StoicNeutron) [inline src/main/java/com/TrackMyWorkout/Server/User/User.java:26]: set to null for safety.

File: build/resources/main/application.properties [hunk 2]
```diff
@@ -2,3 +2,4 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
 spring.datasource.password=1234567890
 spring.jpa.hibernate.ddl-auto=update
+spring.security.enabled=false
```
Comment (StoicNeutron) [inline build/resources/main/application.properties:5]: might not be needed.

**Review by StoicNeutron — APPROVED** [review]
checked.

**Pull request merged**